app/ts/components/subcomponents/ConfigureRpcConnection.tsx

```diff
@@ -21,12 +23,53 @@ const ConfigureRpcContext = createContext<ConfigureRpcContext | undefined>(undef
 const RpcQueryProvider = ({ children }: { children: ComponentChildren }) => {
 	const { value: rpcQuery, waitFor, reset: resetRpcQuery } = useAsyncState<Network>()
 
+	const checkServerAvailability = async (url: string) => {
+		try {
+			const provider = new JsonRpcProvider(url)
+			return await provider.getNetwork()
+		} catch {
+			throw new Error('Server should be reachable')
+		}
+	}
+
+	const validateEthSimulateSupport = async (url: string) => {
+		try {
+			const interimRpc: RpcEntry = { httpsRpc: url, name: '', chainId: 0n, currencyName: '', currencyTicker: '', primary: false, minimized: false, }
+			const requestHandler = new EthereumJSONRpcRequestHandler(interimRpc, true)
+			const service = new EthereumClientService(requestHandler, async () => { }, async () => { })
+			const ethSimulateV1Request: Parameters<typeof service.ethSimulateV1> = [
```

Comment:
what call is this?

Reply:
should send a test eth_simulate request to rpc server? had to compose an interim rpcEntry to fulfill EthereumClientService parameters. do you have another strategy in mind?

Reply:
the code should be changed a bit after [3f215b7](https://github.com/DarkFlorist/TheInterceptor/pull/1030/commits/3f215b732b428d19870cf0d5379878606081e5bf). added console warning for failure to validate. should be helpful for support team